src/validation.cpp

```diff
@@ -4840,6 +4840,20 @@ bool ChainstateManager::ActivateSnapshot(
     return true;
 }
 
+static void FlushSnapshotToDisk(CCoinsViewCache& coins_cache, bool snapshot_loaded)
+{
+    LogPrintf("[snapshot] flushing %s (%.2f MB)... ", /* Continued */
+              snapshot_loaded ? "snapshot chainstate to disk" : "coins cache",
+              coins_cache.DynamicMemoryUsage() / (1000 * 1000));
+
+    const int64_t flush_now{GetTimeMillis()};
+
+    // TODO: if #17487 is merged, add erase=false here if snapshot is loaded, for better performance.
```

Comment:
unrelated: Wouldn't it be better to remove this comment and just do this in `17487`?

Reply:
Yes, done (I removed it in the first push here and then pushed again without it). Made a note in that PR: https://github.com/bitcoin/bitcoin/pull/17487#discussion_r767146237.